<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>PTZSettings</class>
 <widget class="QWidget" name="PTZSettings">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>750</width>
    <height>500</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>PTZ.Settings.Title</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <item>
    <widget class="QTabWidget" name="tabWidget">
     <property name="currentIndex">
      <number>0</number>
     </property>
     <widget class="QWidget" name="generalTab">
      <attribute name="title">
       <string>PTZ.Settings.General</string>
      </attribute>
      <layout class="QVBoxLayout" name="verticalLayout_5">
       <item>
        <widget class="QScrollArea" name="scrollArea">
         <property name="widgetResizable">
          <bool>true</bool>
         </property>
         <widget class="QWidget" name="scrollAreaWidgetContents_3">
          <property name="geometry">
           <rect>
            <x>0</x>
            <y>0</y>
            <width>702</width>
            <height>437</height>
           </rect>
          </property>
          <layout class="QVBoxLayout" name="verticalLayout_4">
           <item>
            <widget class="QCheckBox" name="autoselectCheckBox">
             <property name="toolTip">
              <string>PTZ.Settings.Autoselect.Tooltip</string>
             </property>
             <property name="text">
              <string>PTZ.Settings.Autoselect</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QCheckBox" name="livemoveCheckBox">
             <property name="toolTip">
              <string>PTZ.Settings.LiveMoveLock.Tooltip</string>
             </property>
             <property name="text">
              <string>PTZ.Settings.LiveMoveLock</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QCheckBox" name="speedRampCheckBox">
             <property name="toolTip">
              <string>PTZ.Settings.SpeedRamp.Tooltip</string>
             </property>
             <property name="text">
              <string>PTZ.Settings.SpeedRamp</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QGroupBox" name="joystickGroupBox">
             <property name="title">
              <string>PTZ.Settings.JoystickControl</string>
             </property>
             <property name="checkable">
              <bool>true</bool>
             </property>
             <property name="checked">
              <bool>false</bool>
             </property>
             <layout class="QVBoxLayout" name="verticalLayout_6">
              <item>
               <layout class="QGridLayout" name="gridLayout">
                <item row="0" column="0" colspan="2">
                 <widget class="QListView" name="joystickNamesListView"/>
                </item>
                <item row="1" column="0">
                 <widget class="QLabel" name="joystickSpeedLabel">
                  <property name="text">
                   <string>PTZ.Settings.Speed</string>
                  </property>
                 </widget>
                </item>
                <item row="1" column="1">
                 <widget class="DoubleSlider" name="joystickSpeedSlider">
                  <property name="orientation">
                   <enum>Qt::Orientation::Horizontal</enum>
                  </property>
                 </widget>
                </item>
                <item row="2" column="0">
                 <widget class="QLabel" name="joystickDeadzoneLabel">
                  <property name="text">
                   <string>PTZ.Settings.DeadzoneSize</string>
                  </property>
                 </widget>
                </item>
                <item row="2" column="1">
                 <widget class="DoubleSlider" name="joystickDeadzoneSlider">
                  <property name="toolTip">
                   <string>PTZ.Settings.DeadzoneSlider.Tooltip</string>
                  </property>
                  <property name="orientation">
                   <enum>Qt::Orientation::Horizontal</enum>
                  </property>
                 </widget>
                </item>
               </layout>
              </item>
              <item>
               <layout class="QGridLayout" name="joystickMapGridLayout"/>
              </item>
              <item>
               <layout class="QGridLayout" name="joystickButtonGridLayout"/>
              </item>
             </layout>
            </widget>
           </item>
          </layout>
         </widget>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="camerasTab">
      <attribute name="title">
       <string>PTZ.Cameras</string>
      </attribute>
      <layout class="QHBoxLayout" name="horizontalLayout_3">
       <property name="leftMargin">
        <number>0</number>
       </property>
       <property name="topMargin">
        <number>0</number>
       </property>
       <property name="rightMargin">
        <number>0</number>
       </property>
       <property name="bottomMargin">
        <number>0</number>
       </property>
       <item>
        <widget class="QSplitter" name="splitter">
         <property name="orientation">
          <enum>Qt::Orientation::Horizontal</enum>
         </property>
         <widget class="QWidget" name="widget_1" native="true">
          <layout class="QVBoxLayout" name="verticalLayout_3">
           <property name="sizeConstraint">
            <enum>QLayout::SizeConstraint::SetMinimumSize</enum>
           </property>
           <item>
            <widget class="QLabel" name="label_2">
             <property name="text">
              <string>PTZ.Devices</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QListView" name="deviceList">
             <property name="contextMenuPolicy">
              <enum>Qt::ContextMenuPolicy::CustomContextMenu</enum>
             </property>
             <property name="editTriggers">
              <set>QAbstractItemView::EditTrigger::NoEditTriggers</set>
             </property>
            </widget>
           </item>
           <item>
            <layout class="QHBoxLayout" name="horizontalLayout_2">
             <item>
              <widget class="QPushButton" name="addPTZ">
               <property name="toolTip">
                <string>PTZ.AddDevice.Tooltip</string>
               </property>
               <property name="accessibleName">
                <string>Add</string>
               </property>
               <property name="text">
                <string notr="true"/>
               </property>
               <property name="flat">
                <bool>true</bool>
               </property>
               <property name="class" stdset="0">
                <string notr="true">icon-plus</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QPushButton" name="removePTZ">
               <property name="toolTip">
                <string>PTZ.RemoveDevice.Tooltip</string>
               </property>
               <property name="accessibleName">
                <string>Remove</string>
               </property>
               <property name="text">
                <string notr="true"/>
               </property>
               <property name="flat">
                <bool>true</bool>
               </property>
               <property name="class" stdset="0">
                <string notr="true">icon-trash</string>
               </property>
              </widget>
             </item>
             <item>
              <spacer name="horizontalSpacer_2">
               <property name="orientation">
                <enum>Qt::Orientation::Horizontal</enum>
               </property>
               <property name="sizeHint" stdset="0">
                <size>
                 <width>40</width>
                 <height>20</height>
                </size>
               </property>
              </spacer>
             </item>
            </layout>
           </item>
          </layout>
         </widget>
         <widget class="QWidget" name="widget" native="true">
          <layout class="QVBoxLayout" name="propertiesLayout" stretch="0">
           <property name="sizeConstraint">
            <enum>QLayout::SizeConstraint::SetDefaultConstraint</enum>
           </property>
           <item>
            <layout class="QHBoxLayout" name="horizontalLayout_4">
             <item>
              <spacer name="horizontalSpacer">
               <property name="orientation">
                <enum>Qt::Orientation::Horizontal</enum>
               </property>
               <property name="sizeHint" stdset="0">
                <size>
                 <width>40</width>
                 <height>20</height>
                </size>
               </property>
              </spacer>
             </item>
             <item>
              <widget class="QPushButton" name="applyButton">
               <property name="text">
                <string>Apply</string>
               </property>
              </widget>
             </item>
            </layout>
           </item>
          </layout>
         </widget>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="aboutTab">
      <attribute name="title">
       <string>About</string>
      </attribute>
      <layout class="QVBoxLayout" name="verticalLayout_2">
       <item>
        <widget class="QLabel" name="versionLabel">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Preferred" vsizetype="Minimum">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="alignment">
          <set>Qt::AlignmentFlag::AlignLeading|Qt::AlignmentFlag::AlignLeft|Qt::AlignmentFlag::AlignTop</set>
         </property>
         <property name="openExternalLinks">
          <bool>true</bool>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
   <item>
    <widget class="QLabel" name="labelNdiRegisteredTrademark">
     <property name="text">
      <string>NDI_IS_A_REGISTERED_TRADEMARK_TEXT</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
  </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>DoubleSlider</class>
   <extends>QSlider</extends>
   <header>double-slider.hpp</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
